src/api/routes/status.ts

```diff
@@ -15,6 +15,12 @@ export function createStatusRouter(db: PgStore): express.Router {
         server_version: `stacks-blockchain-api ${API_VERSION.tag} (${API_VERSION.branch}:${API_VERSION.commit})`,
         status: 'ready',
       };
+      const pox1UnlockHeight = await db.getPox1UnlockHeight();
+      if (pox1UnlockHeight.found) {
+        response.pox_v1_unlock_height = pox1UnlockHeight.result;
+      } else {
+        response.pox_v1_unlock_height = null as any;
+      }
```

Comment:
Is this change temporary? i.e. will we need to remove this once it's no longer relevant? Or will it disappear once this height is reached?